6.bulk_Morphology_Elastic_Network/1.train_models/notebooks/1.train_regression_multi_output.ipynb

```diff
@@ -0,0 +1,419 @@
+{
```

Comment:
<p>Confused about this code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Honestly, me too. I couldn't seem to get my types to work. The argparse bool was returning <code>True</code> everytime, so this was my weird work around to just pass a <code>str</code> into through the CLI for my args  </p>